### ISSUE TITLE
False positive for no-store-subscription

### ISSUE THREAD

**Issue by e-oz** [timdeschryver/eslint-plugin-ngrx#175]
## Behavior

It highlights not only subscriptions to the store, but just any subscription if a store is involved.

## Code snippet

```
    this.formCtrlOrg.valueChanges.pipe(takeUntil(this.drop)).subscribe((orgName) => {
      this.store.dispatch(UserPageActions.checkOrgNameInput({orgName}));
    });
```
